docs/book/src/developers/development.md

```diff
@@ -525,6 +525,7 @@ With the following environment variables defined, you can build a CAPZ cluster f
 | `WINDOWS_FLAVOR`     | Optional, can be `containerd` or `containerd-2022`, when not specified dockershim is used |
 | `KUBETEST_WINDOWS_CONFIG`     | Optional, can be `upstream-windows-serial-slow.yaml`, when not specified `upstream-windows.yaml` is used |
 | `WINDOWS_CONTAINERD_URL`     | Optional, can be any url to a `tar.gz` file containing binaries for containerd in the same format as upstream package |
+| `SKIP_E2E_TEST`           | Optional, if `true` then the cluster will be created but the conformance tests will not be run against the cluster.  |
```

Comment:
Nitpick: I think `SKIP_E2E_TESTS` reads better, but this lgtm either way.

Reply:
@jsturtevant are you planning on changing it or is this ready for merge?

Reply:
I will update this if we have agreement we should go with this over a way to provide conformance tests ginkgo parameters to e2e-test binary https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2125#issuecomment-1051262577